package.json

```diff
@@ -135,7 +136,7 @@
     "ts-jest": "26.5.1",
     "ts-loader": "8.0.17",
     "typescript": "4.1.5",
-    "webpack": "5.24.1",
+    "webpack": "^4.46.0",
```

Comment:
Note for reviewer. Components doesn't work well with webpack 5, so reverted to v4. More context in https://github.com/WordPress/gutenberg/issues/27984#issuecomment-754530982

Reply:
Decision was discussed in this thread: p1618406589275200-slack-CGGCLBN58